packages/@storefront/core/test/unit/core/url-beautifier/details/parser.ts

```diff
@@ -97,6 +97,23 @@ suite('DetailsUrlParser', ({ expect }) => {
     expect(parsed.variants).to.eql(expectedDetail.variants);
   });
 
+  // tslint:disable-next-line max-line-length
+  it('should extract query string parameters of a URL specifying "area" and "collection", and add them to the return object', () => {
```

Comment:
I would split this into two tests, one for area and one for collection.

Reply:
Will do.